deploy/charts/rook-ceph/values.yaml

```diff
@@ -645,11 +647,11 @@ enableOBCWatchOperatorNamespace: true
 # -- Set tolerations and nodeAffinity [^1] for admission controller pod.
 # The admission controller would be best to start on the same nodes as other ceph daemons.
 admissionController:
-  # tolerations:
-  #    - key: key
-  #      operator: Exists
-  #      effect: NoSchedule
-  # nodeAffinity: key1=value1,value2; key2=value3
+# tolerations:
```

Comment:
Can you revert these spacing changes? Or if the editor is being a nuisance, you could instead replace the spaces to be after the `#`.

Reply:
Ah, that was an oversight, my editor was being "helpful" 😅 
I'll revert that.